app/src/main/java/com/tasomaniac/openwith/settings/SettingsFragment.java

```diff
@@ -226,11 +227,10 @@ private void displayLicensesDialogFragment() {
     }
 
     private void startContactEmailChooser() {
-        ShareCompat.IntentBuilder.from(getActivity())
-                .addEmailTo("Said Tahsin Dane <tasomaniac+openlinkwith@gmail.com>")
-                .setSubject(getString(R.string.app_name))
-                .setType("message/rfc822")
-                .startChooser();
+        EmailIntentBuilder.from(getActivity())
+                .to("tasomaniac+openlinkwith@gmail.com")
```

Comment:
But my app is not about anything email. This is just lazy and easy contact me implementation. 
I don't want to change something that already works great for me. My name there is very valuable for me because I make personal contact with my users.